tests/test_code_references.py

```diff
@@ -315,36 +310,26 @@ def predict(resize):
         plt.show,
         ReferenceProxy("int"),
         ReferenceProxy("len"),
+        # These are repeated since we are deduping using object ids.
```

Comment:
What if we dedup using hash when available for a reference? That might catch some of these duplicated references. Any reference that is not hashable can still be deduped using the id like what I'm doing here. What do you all think?

Reply:
Hmm... if the hasher is already maintaining an internal cache already-hashed objects, how important is it to remove duplicate references at all?

Reply:
Not a lot tbh. I don't mind just killing this PR (or merge this PR with only removing the TODO).